packages/http-server/src/serialize.ts

```diff
@@ -33,3 +33,20 @@ export default [
     },
   },
 ];
+
+export const serialize = (payload: unknown, contentType?: string) => {
```

Comment:
This function does 3 things:
1) return `undefined`
2) throws an exception
3) serializes with a found serializer

If we need to find a serializer, let's use `Option` at least. Let's not throw when a serializer is not found.
Also, is it possible to do `reply.send(undefined)`?

@XVincentX, WDYT?

Reply:
@lag-of-death gave me a nice lesson on what are those tiny FP pieces about.

However I feel like it is not only about that function, server.ts should be also changed to be compliant with FP approach for the whole thing to make sense (because of throws and nulls).

Reply:
Correct, refactoring this is not a 5 minute task. You'll get you hands dirty soon, so I'm not concerned about it. We're good.